doc/man/scons.xml

```diff
@@ -104,6 +104,10 @@ in a future &SCons; release.
 The CPython project retired 3.6 in Sept 2021:
 <ulink url="https://peps.python.org/pep-0494"/>.
 </para>
+<para>
+<emphasis>Changed in version 4.9.0:</emphasis>
```

Comment:
Change 4.9.0 to `NEXT_RELEASE` we search and replace this prior to release with whatever release version it ends up being.